src/App.tsx

```diff
@@ -1,26 +1,61 @@
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useDispatch } from 'react-redux';
```

Comment:
```
export const useAppDispatch = useDispatch<AppDispatch>;
```

Consider using `useAppDispatch` if u have a custom hook for dispatching actions. This can help ensure type safety when dispatching actions in a TypeScript environment.
U are already have it in `store.ts`

Use it in all such cases